src/iosched/unified.c

```diff
@@ -1772,6 +1955,10 @@ ssize_t _unified_insert_new_request(const char *buf, off_t offset, size_t count,
 		releaseread_mrsw(&priv->lock);
 		return -LTFS_NO_MEMORY;
 	}
+
+	_unified_get_dentry_priv(d, &dpr, priv);
```

Comment:
Does it matter if we get a NULL `dpr` here or not? We're not checking that; not sure if we should.

Reply:
It's not needed actually. I will remove this initializer for now.  Fixed in my branch.

But we may need to add some initializer in the future because some compilers reports a warning.